### PR TITLE
add dhcpcd

### PR DESCRIPTION
**Pull request by wlan0**
@ibuildthecloud

### CODE REVIEW

**Comment by ibuildthecloud** [comment]
Can you remove dhclient?

**Comment by wlan0** [comment]
done.

File: config/buildroot-config [hunk 1]
```diff
@@ -1514,8 +1514,8 @@ BR2_PACKAGE_READLINE=y
 BR2_PACKAGE_DHCP=y
```
Comment (ibuildthecloud) [inline config/buildroot-config]: This needs to be unset too

**Comment by wlan0** [comment]
@ibuildthecloud This is ready

**Comment by wlan0** [comment]
@ibuildthecloud  This is ready

**Comment by masaeedu** [comment]
@wlan0 What is the recommended approach for releasing and registering DHCP leases? It's weird, but  I'm getting a new IP address every time I reboot the machine, yet no DHCPRELEASE is being issued when the machine goes down.

**Comment by segfly** [comment]
@masaeedu I think I found the root cause of getting a new IP on every reboot. Please comment on rancher/os#909 if you would like to see it addressed.

**Pull request merged**